pom.xml

```diff
@@ -20,12 +33,13 @@
         <scala.binary.version>2.11</scala.binary.version>
         <scala.version>2.11</scala.version>
     </properties>
+
+    <modules>
+        <module>tikv-client</module>
```

Comment:
shall we put integration test here as well?

Reply:
DONE, push commit later.